sentence_transformers/SentenceTransformer.py

```diff
@@ -1332,6 +1365,8 @@ def _load_sbert_model(
                     kwargs["model_args"].update(hub_kwargs)
                 else:
                     kwargs["model_args"] = hub_kwargs
+                kwargs["model_args"].update(model_args or {})
```

Comment:
```suggestion
                if model_args:
                    kwargs["model_args"].update(model_args)
```
nit: I think this is slightly cleaner than calling `update` with an empty dict